app/javascript/mastodon/features/home_timeline/index.jsx

```diff
@@ -195,7 +195,7 @@ class HomeTimeline extends PureComponent {
     }
 
     return (
-      <Column bindToDocument={!multiColumn} ref={this.setRef} label={intl.formatMessage(messages.title)}>
+      <Column bindToDocument={!multiColumn} innerRef={this.setRef} label={intl.formatMessage(messages.title)}>
```

Comment:
`innerRef`?

Reply:
I am not sure why I did this 🤔 

Reverted, `ref` is the one to use here